packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/index.md

```diff
@@ -83,7 +83,7 @@ The Identity Engine upgrade documentation below discusses the various upgrade ta
 
 * [Planning embedded auth app upgrades](/docs/guides/oie-upgrade-planning-embedded-upgrades/): Learn about how to embark on the upgrade process, the steps required, and how you can stage your upgrade. Also discussed are suggestions on when to test and how to roll out the application upgrades to your users.
 
-* Upgrade authentication services and applications to use the Interaction Code grant type: Learn how to configure your embedded applications and the corresponding authorization servers to use the Interaction Code grant type. More detailed information coming soon. <!-- (/docs/guides/implement-grant-type/authcode/) -->
+* [Upgrade authentication services and applications to use the Interaction Code grant type](/docs/guides/implement-grant-type/interactioncode/main/#set-up-your-authorization-server): Learn how to configure your embedded applications and the corresponding authorization servers to use the Interaction Code grant type.
```

Comment:
For some reason, when I run a local build, this link insists on going to the #grant-type-flow anchor.